core/templates/dev/head/pages/profile/profile.html

```diff
@@ -416,7 +411,10 @@ <h2 class="oppia-profile-username-small-screen">
       top: 50%;
       transform: translate(-50%, -50%);
     }
-
+    .oppia-navigation-button.disabled{
```

Comment:
Please follow the existing style in this file carefully: add space before {.

/cc @apb7

Reply:
Changed.!

Reply:
@seanlip: Added to #5097. Thanks!